orchestra/utils/task_lifecycle.py

```diff
@@ -133,6 +133,26 @@ def is_worker_certified_for_task(
     return certified_for_task
 
 
+def get_role_from_counter(role_counter):
+    if role_counter == 0:
+        return WorkerCertification.Role.ENTRY_LEVEL
+    return WorkerCertification.Role.REVIEWER
+
+
+def role_counter_required_for_new_task(task):
+    if task.status not in [Task.Status.AWAITING_PROCESSING,
+                           Task.Status.PENDING_REVIEW]:
+        raise TaskAssignmentError('Status incompatible with new assignment')
+    assignments_count = task.assignments.count()
```

Comment:
why not:

`return task.assignments.fitler(assignment_counter__lte=assignments_count).order_by('assignment_counter').first().assignment_counter`